client/src/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import QuestionsTab from './QuestionsTab.jsx';
```

Comment:
You can remove .jsx after you add  " resolve: { extensions: ['.js', '.jsx'] }, " in your webpack.config.js. Since it is recommended by airbnb style guide, it will be better for us to follow the guide in the future code or refactor it.